src/main/java/domain/RoundResult.java

```diff
@@ -1,20 +1,47 @@
 package domain;
 
+import view.RoundResultFormatter;
+
 import java.util.ArrayList;
 import java.util.List;
 
 public class RoundResult {
     List<RacingCarCurrentStatus> raceProgress;
 
-    public RoundResult(List<RacingCar> cars) {
-        this.raceProgress = new ArrayList<>();
-        for (RacingCar car : cars) {
-            RacingCarCurrentStatus currentStatus = car.getCurrentStatus();
-            raceProgress.add(currentStatus);
+    public RoundResult(RacingCars cars) {
+        this.raceProgress = cars.listCurrentRacingCarStatuses();
+    }
+
+    public List<String> findWinners() {
+        List<String> winners = new ArrayList<>();
+        for (RacingCarCurrentStatus car : raceProgress) {
+            addIfWinner(car, findMaxPosition(), winners);
+        }
+        return winners;
+    }
+
+    public RoundResultFormatter generateFormatter() {
+        return new RoundResultFormatter(raceProgress);
+    }
+
+    private Position findMaxPosition() {
+        Position maxPosition = new Position(0);
+        for (RacingCarCurrentStatus car : raceProgress) {
+            maxPosition = updateMaxPosition(car, maxPosition);
+        }
+        return maxPosition;
+    }
+
+    private Position updateMaxPosition(RacingCarCurrentStatus carStatus, Position maxPosition) {
+        if (carStatus.isAheadOf(maxPosition)) {
+            maxPosition = carStatus.position();
```

Comment:
```suggestion
            return carStatus.position();
```

값을 변경할 필요는 없어보이네요 ㅎ
그렇다면, `updateMaxPosition`의 이름도 조금 어색해 보이네요 🤔

Reply:
```java
private Position updateMaxPosition(RacingCarCurrentStatus carStatus, Position maxPosition) {
    if (carStatus.isAheadOf(maxPosition)) {
        maxPosition = carStatus.position();
    }
    return maxPosition;
}
```
`update`의미가 필요한가요?
`MaxPosition` 의미가 필요한가요?

호출된 곳에서의 `Max`라는 의미가 연쇄적으로 불필요하게 전파되고 있는게 아닐지 고민해보아도 좋을것 같습니다. 😄